src/fabric/test/fabric2_db_misc_tests.erl

```diff
@@ -111,3 +113,24 @@ is_system_db({DbName, Db, _}) ->
 ensure_full_commit({_, Db, _}) ->
     ?assertEqual({ok, 0}, fabric2_db:ensure_full_commit(Db)),
     ?assertEqual({ok, 0}, fabric2_db:ensure_full_commit(Db, 5)).
+
+
+metadata_bump({DbName, _, _}) ->
```

Comment:
We should probably add a similar test for the db_version changing as well

Reply:
Added a test for db version bump